node/miden-node-example.toml

```diff
@@ -0,0 +1,19 @@
+[block_producer]
+store_url = "http://localhost:28943"
+
+[block_producer.endpoint]
+host = "localhost"
+port = 48046
```

Comment:
nit: I'd prefer a more compact representation - similar to what we have on line 11 below. Or at least we should be consistent.

Also, here and below, could we add comments explaining the choice of port numbers? (I think we had such comments in the code).

Reply:
> nit: I'd prefer a more compact representation - similar to what we have on line 11 below. Or at least we should be consistent.

> Also, here and below, could we add comments explaining the choice of port numbers? (I think we had such comments in the code).

Since this is an example file, is there actual value in how we came up with that number? IMO, any ephemeral port number would do, and whatever algorithm we used to generate it is irrelevant. Unless I'm missing something?

Reply:
Yeah - I'm fine not including comments - though, I think it is cool info to include mostly for historical reasons.

Reply:
added them - we can always remove in the future